docs/platform/names/about-dpns.md

```diff
@@ -0,0 +1,9 @@
+## What is DPNS
+
+DPNS is a special Dash Platform Application that is intended to provide a naming service for the Application Chain.  
+
+Decoupling from the Blockchain identity allow to provide a unique user experience coupled with the highest security while remaining compatible with Identity Standard.
```

Comment:
I think you were referencing DiDs here. If not, ignore this :slightly_smiling_face: 
```suggestion
Decoupling name from the blockchain identity enables a unique user experience coupled with the highest security while remaining compatible with [Decentralized Identifiers](https://www.w3.org/TR/did-core/).
```